tests-behat/bootstrap/Context.php

```diff
@@ -450,7 +450,7 @@ protected function getFinishedScript(): string
         return 'document.readyState === \'complete\''
             . ' && typeof $ !== \'undefined\' && jQuery.active === 0'
             . ' && typeof atk !== \'undefined\' && atk.vueService.areComponentsLoaded()'
-            . ' && 0 === jQuery(\':animated\').length)';
+            . ' && 0 === jQuery(\':animated\').length';
```

Comment:
I think this is not needed with `$.fx.off = true;`, as JQuery JS animations are disabled

Reply:
understand but there is also no guarantee that a Fomantic module is not setting it back to false when initializing.

Reply:
unles it does - remove, because it is very unexpected it does so

Reply:
why remove? this is just a safety test. besides, all seem to be working well now. The reason I added this test was that when some test was failing, it was because it could not find one of the dropdown control, sometimes it was the first one, sometimes it was the second one. Adding this test seems to fix the issue.

Reply:
> Adding this test seems to fix the issue.

I am doing testing now, if this is needed, of course, it will stay. Otherwise I think it is duplicated to the jQuery setting as libs that set it basically all use jQuery :)